indico/modules/categories/controllers/display.py

```diff
@@ -18,13 +18,14 @@
 
 from datetime import datetime, timedelta
 from io import BytesIO
-from itertools import groupby
+from itertools import chain, groupby
 from math import ceil
-from operator import attrgetter
+from operator import attrgetter, itemgetter
 
 from dateutil.relativedelta import relativedelta
 from flask import jsonify, request, session
 from sqlalchemy.orm import joinedload, load_only, subqueryload, undefer
+from sqlalchemy.sql.expression import and_, or_
```

Comment:
`db.and_`, `db.or_` could also be used instead of importing them directly, either way is fine though.